### PR TITLE
[83] Rename packages to be lowercase

### PR DESCRIPTION
**Pull request by sam-i-am012**

### CODE REVIEW

**Review by korydraughn — COMMENTED** [review]
Nothing unusual stands out to me.

Remove "current" from the commit message and this will be ready.

Do the tests pass?

**Comment by sam-i-am012** [comment]
Yep all tests pass, I've edited the commit message.

**Review by korydraughn — APPROVED** [review]
Pound it.

**Comment by sam-i-am012** [comment]
Done.

**Comment by korydraughn** [comment]
Github is reporting several conflicts. Please resolve.

**Comment by sam-i-am012** [comment]
This PR may be deleted because I think I messed something up with rebase and I think it'll be easier if I switch over to my backup of this branch and start fresh.

**Comment by sam-i-am012** [comment]
Okay, I resolved the conflicting changes. It was mostly due to me adding files in `main` from the previous PR that was merged where the packages where still capitalized so all I had to do change them to lowercase. All tests pass.

**Comment by sam-i-am012** [comment]
Okay, I resolved the conflicting changes. It was mostly due to me adding files in main from the previous PR that was merged where the packages where still capitalized so all I had to do change them to lowercase. All tests pass.

File: src/test/java/org/irods/operations/UserGroupOperationsTest.java [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 import org.irods.IrodsHttpClient;
-import org.irods.Serialize.ModifyMetadataOperations;
-import org.irods.Util.MetadataOperation;
-import org.irods.Util.Response;
-import org.irods.Util.UserType;
-import org.junit.Before;
+import org.irods.serialize.ModifyMetadataOperations;
+import org.irods.util.Response;
+import org.irods.util.MetadataOperation;
+import org.irods.util.UserType;import org.junit.Before;
```
Comment (korydraughn) [inline src/test/java/org/irods/operations/UserGroupOperationsTest.java:9]: Put second `import` on its own line.

Comment (sam-i-am012) [reply src/test/java/org/irods/operations/UserGroupOperationsTest.java]: Oops, I added it to its own line.

**Comment by sam-i-am012** [comment]
All existing review comments have been resolved.

**Comment by korydraughn** [comment]
If the tests pass and you're happy with it, squash it to one commit and we'll get it merged.

**Comment by sam-i-am012** [comment]
Done, all tests pass.

**Comment by korydraughn** [comment]
Merging.

**Pull request merged**